spring-cloud-starter-stream-processor-aggregator/README.adoc

```diff
@@ -6,6 +6,30 @@ Use the `aggregator` application to combine multiple messages into one, based on
 This processor is fully based on the Aggregator component from http://docs.spring.io/spring-integration/reference/html/messaging-routing-chapter.html#aggregator[Spring Integration].
 So, please, consult there for use-cases and functionality.
 
+== Input
+
+=== Headers
+
+If the aggregation and correlation logic is based on the default strategies, the `correlationId`, `sequenceNumber` and `sequenceSize` headers must be presented in the incoming message.
+Otherwise no any headers requirements.
```

Comment:
M-m-m. Well, actually I meant like:
> you don't need any headers at all if you don't rely on the default strategies

Would be glad to rephrase to make it clear for everybody.

Thanks for the pointer!